lib/Model/SelectHelper.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020 Joas Schilling <coding@schilljs.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\Model;
+
+use OCP\DB\QueryBuilder\IQueryBuilder;
+
+class SelectHelper {
+	public function selectRoomsTable(IQueryBuilder $query, string $alias = 'r'): void {
+		if ($alias !== '') {
+			$alias .= '.';
+		}
+
+		$query->addSelect($alias . 'type')
+			->addSelect($alias . 'read_only')
+			->addSelect($alias . 'lobby_state')
+			->addSelect($alias . 'sip_enabled')
+			->addSelect($alias . 'assigned_hpb')
+			->addSelect($alias . 'token')
+			->addSelect($alias . 'name')
+			->addSelect($alias . 'description')
+			->addSelect($alias . 'password')
+			->addSelect($alias . 'active_guests')
+			->addSelect($alias . 'active_since')
+			->addSelect($alias . 'last_activity')
+			->addSelect($alias . 'last_message')
+			->addSelect($alias . 'lobby_timer')
+			->addSelect($alias . 'object_type')
+			->addSelect($alias . 'object_id')
+			->addSelect($alias . 'listable')
+			->selectAlias($alias . 'id', 'r_id');
```

Comment:
r_id should take the "r" from the alias, for consistency

Reply:
No, it needs to be `r_id` to match https://github.com/nextcloud/spreed/blob/70679077ca966f48e5dd710b75613b090742e915/lib/Manager.php#L178